tools/clang/unittests/HLSLExec/LongVectorTestData.h

```diff
@@ -0,0 +1,241 @@
+#pragma once
```

Comment:
Should I be using ifdef include guards instead of #pragma once?

Reply:
I guess so :(  https://llvm.org/docs/CodingStandards.html#header-guard